src/ts/components/account/SelectNetwork.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import t from '../../services/i18n'
+import Progress from './Progress'
+import { IAppState } from '../../background/store/all'
+import { withRouter, RouteComponentProps } from 'react-router'
+import { connect } from 'react-redux'
+import {
+  Button as StyledButton,
+  ContentContainer,
+  TopSection
+} from '../basic-components'
+import { setLocked, setCreated, setNewPhrase } from '../../background/store/wallet'
+import { setError } from '../../background/store/error'
+import { saveSettings } from '../../background/store/settings'
+import { HOME_ROUTE } from '../../constants/routes'
+import { Divider, Form, Input } from 'semantic-ui-react'
+import styled from 'styled-components'
+import NetworkList from './NetworkList'
+
+interface ISelectNetworkProps extends RouteComponentProps, StateProps, DispatchProps {}
+
+class SelectNetwork extends React.Component<ISelectNetworkProps> {
+
+  gotoDashboard = () => {
+    this.props.history.push(HOME_ROUTE)
+  }
+
+  render () {
+
+    return (
+      <ContentContainer>
+        <TopSection>
+          <Progress step={3}/>
+        </TopSection>
+        <Form>
+          <ChainInput placeholder={'Chain Search'}/>
+        </Form>
+        <NetworkSection>
+          <NetworkList/>
+        </NetworkSection>
+        <ChainQuote>Start by selecting <span>3 chains</span></ChainQuote>
+        <Divider />
+        <StyledButton type='button' onClick={this.gotoDashboard}>
+          {t('selectNetwork')}
+        </StyledButton>
+      </ContentContainer>
+    )
+  }
+}
+
+const mapStateToProps = (state: IAppState) => {
+  return {
+    settings: state.settings,
+    color: state.settings.color
+  }
+}
+
+const mapDispatchToProps = { saveSettings, setLocked, setCreated, setError, setNewPhrase }
```

Comment:
You probably don't need to map these props

Reply:
I left it out just in case for interacting with backend(Adding network in redux, etc) but I do not see valid reason for using it now. So I will remove it.

Reply:
For now as just an UI I agree that we do not need these, but we might need it after clicking `select and start` button because the selected networks for the account need to be saved.